musicbrainzngs/mbxml.py

```diff
@@ -775,10 +775,17 @@ def make_tag_request(**kwargs):
                 e_xml = ET.SubElement(e_list, "{%s}%s" % (NS, entity_type.replace('_', '-')))
                 e_xml.set("{%s}id" % NS, e)
                 taglist = ET.SubElement(e_xml, "{%s}user-tag-list" % NS)
-                for tag in tags:
-                    usertag_xml = ET.SubElement(taglist, "{%s}user-tag" % NS)
-                    name_xml = ET.SubElement(usertag_xml, "{%s}name" % NS)
-                    name_xml.text = tag
+                if isinstance(tags, dict):
+                    for tag, vote in tags.items():
+                        usertag_xml = ET.SubElement(taglist, "{%s}user-tag" % NS)
+                        usertag_xml.set('vote', vote)
```

Comment:
we should raise an error if the `vote` parameter isn't a valid value

Reply:
I implemented a ValueError